plugins/auth-backend/src/providers/aws-alb/provider.ts

```diff
@@ -25,73 +27,174 @@ import { KeyObject } from 'crypto';
 import { Logger } from 'winston';
 import NodeCache from 'node-cache';
 import { JWT } from 'jose';
-import { CatalogApi } from '@backstage/catalog-client';
+import { TokenIssuer } from '../../identity/types';
+import { CatalogIdentityClient } from '../../lib/catalog';
+import { Profile as PassportProfile } from 'passport';
+import { makeProfileInfo } from '../../lib/passport';
 
 const ALB_JWT_HEADER = 'x-amzn-oidc-data';
-/**
- * A callback function that receives a verified JWT and returns a UserEntity
- *  @param {payload} The verified JWT payload
- */
-type AwsAlbAuthProviderOptions = {
+const ALB_ACCESSTOKEN_HEADER = 'x-amzn-oidc-accesstoken';
+
+type Options = {
   region: string;
   issuer?: string;
-  identityResolutionCallback: ExperimentalIdentityResolver;
```

Comment:
Yep I think this is fine to remove in a `patch`, prolly get rid of the type too as it was only used here

Reply:
I will write up a changelog for this then. There is a [document](https://github.com/backstage/backstage/blob/master/contrib/docs/tutorials/aws-alb-aad-oidc-auth.md) in `contrib` as well. Not sure how you think about things in contrib, should I update that as well?